pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -651,6 +661,14 @@ func (r *RepoSyncReconciler) mapSecretToRepoSyncs(secret client.Object) []reconc
 	return requests
 }
 
+func (r *RepoSyncReconciler) mapAdmissionWebhookToRepoSyncs(admissionWebhook client.Object) []reconcile.Request {
+	sRef := client.ObjectKeyFromObject(admissionWebhook)
+	if sRef.Name == "admission-webhook.configsync.gke.io" {
```

Comment:
You can use the constant in https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/webhook/configuration/constants.go#L30.

Reply:
Done